src/routes/(app)/_components/layout/navbar.svelte

```diff
@@ -9,9 +18,17 @@
     </a>
   </div>
   <div class="flex justify-end gap-7">
-    <a href="#/">Equipa</a>
-    <a href="#/">Projetos</a>
-    <a href="#/">Eventos</a>
-    <a href="#/">Contactos</a>
+    <a class:active={currentUrl === "http://localhost:3000/#/team/"} href="#/team/"><p class = "font-bold">Equipa</p></a>
+    <a class:active={currentUrl === "http://localhost:3000/#/projects/"} href="#/projects/"><p class = "font-bold">Projetos</p></a>
+    <a class:active={currentUrl === "http://localhost:3000/#/events/"} href="#/events/"><p class = "font-bold">Eventos</p></a>
+    <a class:active={currentUrl === "http://localhost:3000/#/contacts/"} href="#/contacts/"><p class = "font-bold">Contactos</p></a>
```

Comment:
The `#` notation is used to reference an id of a HTML element inside a page, so this is not the intended use case! The previous one was a placeholder :) If you actually wanted to use it, you could do `/contacts/#button` (fictional example), so that once you clicked it you'd be redirectioned to the contacts page and scrolled down automatically to the button. Here you should do `/contacts`, `/team` etc. though!

Reply:
Done!